### ISSUE TITLE
CI fails on odoc install

### ISSUE THREAD

**Issue by MoOx** [reasonml-community/bs-react-native#311]
Lot's of PRs are failing because of this problem:

```
[odoc] http://github.com/ocaml-doc/odoc.git updated
[odoc] Installing new package description from
http://github.com/ocaml-doc/odoc.git

odoc needs to be installed.
Your request can't be satisfied:
  - odoc.dev is not available because the package is pinned to http://github.com/ocaml-doc/odoc.git, version dev.

No solution found, exiting
[NOTE] Pinning command successful, but your installed packages may be out of
       sync.
Exited with code 3
```

I tried the things found in the circleci config, and it's works for me.
I am new to opam & co so I am not really able to fix this by my own. Help appreciated.

**Comment by MoOx** [comment]
Feel free to review my attempt to fix this issue in #310

**Comment by MoOx** [comment]
Closed by e9ae1ea
